### PR TITLE
Whitelist metrics for MCG addon

### PR DESCRIPTION
**Pull request by rexagod**
Whitelist metrics for MCG addon, based on following document:
* https://docs.google.com/document/d/19JST--IQyxTa_330D7mjJZUTEi2dqJ9NXcJ9J0SrhyY/edit?usp=sharing

This would enable us to create a dashboard in staging and production
telemeter datahub environments to realize our achieved SLIs in the defined SLO
range.

SLI metrics required by the MCG addon:
* NooBaa_num_buckets_claims
* NooBaa_num_namespace_buckets
* NooBaa_num_unhealthy_bucket_claims
* NooBaa_num_unhealthy_namespace_buckets
* noobaa_accounts_num
* noobaa_total_usage
* odf_system_health_status
* odf_system_iops_total_bytes
* odf_system_throughput_total_bytes

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

### CODE REVIEW

File: configuration/telemeter/metrics.json [hunk 1]
```diff
@@ -79,9 +79,16 @@
...
+  "{__name__=\"NooBaa_num_buckets_claims\"}",
+  "{__name__=\"NooBaa_num_namespace_buckets\"}",
+  "{__name__=\"NooBaa_num_unhealthy_bucket_claims\"}",
+  "{__name__=\"NooBaa_num_unhealthy_namespace_buckets\"}",
   "{__name__=\"noobaa_accounts_num\"}",
   "{__name__=\"noobaa_total_usage\"}",
   "{__name__=\"nto_custom_profiles:count\"}",
+  "{__name__=\"odf_system_health_status\"}",
+  "{__name__=\"odf_system_iops_total_bytes\"}",
+  "{__name__=\"odf_system_throughput_total_bytes\"}",
```
Comment (sabose) [inline configuration/telemeter/metrics.json:91]: Are the above needed for MCG too?

Comment (rexagod) [reply configuration/telemeter/metrics.json:91]: In a word, yes.

So the thing is, these metrics are, in a way, misleading. These are prefixed with `odf_`, but in fact, they don't have any dependency on it. You can see [here](https://github.com/noobaa/noobaa-mixins/blob/d0b6aa9eda0174fc721784cea48fa9e99014e061/rules/rules.libsonnet#L65) that these are just mixins defined and coming from Noobaa itself. It's just that the recording rules that are defined there have been named this way.

We need these to determine our availability (`odf_system_health_status`) and volumetric (`odf_system_iops_total_bytes` and `odf_system_throughput_total_bytes`) SLIs as defined in this table: https://docs.google.com/document/d/19JST--IQyxTa_330D7mjJZUTEi2dqJ9NXcJ9J0SrhyY/edit#heading=h.m0h94blwod4y

**Review by matej-g — APPROVED** [review]
Looks good now, thanks @rexagod

**Comment by simonpasquier** [comment]
I think that the PR should to be reverted since we want https://github.com/openshift/cluster-monitoring-operator/pull/1700 be merged first then rhobs/configuration can be synced from CMO's master branch (with `make whitelisted_metrics`).

**Pull request merged**